StyleCop.Analyzers/StyleCop.Analyzers/StyleCop.Analyzers.csproj

```diff
@@ -37,4 +37,19 @@
     <None Include="**\*.json" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
:memo: The changes to this project file were triggered by a bug in the project system, and can be manually reverted after making any desired changes to the .resx file(s).